data/global.lua

```diff
@@ -23,15 +23,41 @@ function getDistanceBetween(firstPosition, secondPosition)
 	return posDif
 end
 
+function getWorldTime()
+	local realSeconds = tonumber(os.date("%S"))
+	local realMinutes = tonumber(os.date("%M"))
+
+	local worldMinutes = math.floor((realSeconds + (realMinutes * 60)) / 2.5)
+	return worldMinutes
+end
+
 function getFormattedWorldTime()
-	local worldTime = getWorldTime()
-	local hours = math.floor(worldTime / 60)
+	local hoursPrefix = ""
+	local minutesPrefix = ""
+
+	local worldMinutes = getWorldTime()
+	local worldHours = math.floor(worldMinutes/60)
+	
+	worldMinutes = worldMinutes%60
 
-	local minutes = worldTime % 60
-	if minutes < 10 then
-		minutes = '0' .. minutes
+	if worldHours < 10 then hoursPrefix = "0" end
+	if worldMinutes < 10 then minutesPrefix = "0" end
+
+	return hoursPrefix..worldHours..":"..minutesPrefix..worldMinutes
```

Comment:
`string.format("%02d:%02d", worldHours, worldMinutes)` ?
Not sure if we have this function available.